src/pages/user/my-profile/my-profile.scss

```diff
@@ -31,6 +31,39 @@
   }
 }
 
+.my-profile-page-container-without-grid-template {
+  @apply grid;
+  // grid-template-columns: 25% 75%;
+  height: calc(100vh - 160px);
+
+  .my-profile-side-bar {
+    @apply bg-tertiary-300;
+  }
+
+  .my-profile-page {
+    @apply grid h-full w-max gap-3 p-3;
+
+    grid-template-columns: 33% 33% 33%;
```

Comment:
you can use the tailwind class grid-cols-3, it's equivalent to  grid-template-columns: 1fr 1fr 1fr;